setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+
+def readme():
+    with open('README.md') as f:
+        return f.read()
+
+setup(
+    name='sequence-classification',
+    version='0.1',
+    description='module for comparing sequences classifiers',
```

Comment:
Module for comparing sequence classifier.
Wielka litera na początku + `sequences` -> `sequence`.